test/jasmine/tests/parcoords_test.js

```diff
@@ -8,7 +8,7 @@ var attributes = require('@src/traces/parcoords/attributes');
 var createGraphDiv = require('../assets/create_graph_div');
 var delay = require('../assets/delay');
 var destroyGraphDiv = require('../assets/destroy_graph_div');
-var fail = require('../assets/fail_test');
```

Comment:
FYI `fail` is a jasmine global, we were overloading it here, so among other things if you *don't* bring in `assets/fail_test` but you thought you had, linters won't catch it. We should probably change that in the rest of our test suites too...